string command
float32 strength